srcpkgs/lsp-plugins/template

```diff
@@ -1,26 +1,55 @@
 # Template file for 'lsp-plugins'
 pkgname=lsp-plugins
-version=1.2.14
-revision=2
+version=1.2.19
+revision=1
 build_style=gnu-makefile
 make_use_env=yes
 make_build_args="VERBOSE=0"
 hostmakedepends="pkg-config php lv2"
 makedepends="lv2 libglvnd-devel jack-devel libsndfile-devel
- libXrandr-devel cairo-devel freetype-devel"
+ libXrandr-devel cairo-devel freetype-devel gst-plugins-base1-devel"
 depends="lsp-plugins-libs lsp-plugins-xdg
- lsp-plugins-clap lsp-plugins-ladspa lsp-plugins-lv2 lsp-plugins-vst2"
+ lsp-plugins-clap lsp-plugins-ladspa lsp-plugins-lv2 lsp-plugins-vst2
+ lsp-plugins-gst lsp-plugins-vst3"
 short_desc="Collection of open-source audio plugins for studios"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="https://lsp-plug.in/"
 changelog="https://raw.githubusercontent.com/sadko4u/lsp-plugins/master/CHANGELOG"
 distfiles="https://github.com/sadko4u/lsp-plugins/releases/download/$version/lsp-plugins-src-${version}.tar.gz"
-checksum=1a3359ee8b8a82971bd7e9eeabe43f58ce6b4a4793d85fb16f97b100e4edee9a
-nocross="Build system doesn't support cross-compilation properly"
+checksum=f0536d790fada3d896670683b71a2916a7bef55a2395f314370015535ec8d959
+
+if [ "$CROSS_BUILD" ]; then
+	hostmakedepends+=" ${makedepends}"
+fi
 
 do_configure() {
-	make LD="$CXX" PREFIX=/usr config
+	if [ "$CROSS_BUILD" ]; then
+		make config \
+			LD="$CXX" \
+			PREFIX="/usr" \
+			HOST_CC="cc" \
+			HOST_CXX="g++" \
+			HOST_LD="g++" \
+			ARCHITECTURE="XBPS_TARGET_MACHINE" \
+			CROSS_COMPILE="1" \
+			HOST_ARCHITECTURE_CXXFLAGS="$XBPS_CXXFLAGS" \
+			HOST_ARCHITECTURE_CFLAGS="$XBPS_CFLAGS" \
+			HOST_ARCHITECTURE_LDFLAGS="$XBPS_LDFLAGS" \
+			ARCHITECTURE_CXXFLAGS="$CXXFLAGS" \
+			ARCHITECTURE_CFLAGS="$CFLAGS" \
+			ARCHITECTURE_LDFLAGS="$LDFLAGS"
+	else
+		make config \
+			LD="$CXX" \
+			PREFIX="/usr" \
+			HOST_ARCHITECTURE_CXXFLAGS="$CXXFLAGS" \
+			HOST_ARCHITECTURE_CFLAGS="$CFLAGS" \
+			HOST_ARCHITECTURE_LDFLAGS="$LDFLAGS" \
+			ARCHITECTURE_CXXFLAGS="$CXXFLAGS" \
+			ARCHITECTURE_CFLAGS="$CFLAGS" \
+			ARCHITECTURE_LDFLAGS="$LDFLAGS"
+	fi
```

Comment:
```suggestion
	_configure_args=(PREFIX=/usr LD="$CXX" ARCHITECTURE_CFLAGS="$CFLAGS"
	 ARCHITECTURE_CXXFLAGS="$CXXFLAGS" ARCHITECTURE_LDFLAGS="$LDFLAGS")

	if [ "$CROSS_BUILD" ]; then
		_configure_args+=(CROSS_COMPILE=1 HOST_CC=cc HOST_CXX=c++ HOST_LD=c++
		 ARCHITECTURE="$XBPS_TARGET_MACHINE" HOST_ARCHITECTURE_CFLAGS="$XBPS_CFLAGS"
		 HOST_ARCHITECTURE_CXXFLAGS="$XBPS_CXXFLAGS" HOST_ARCHITECTURE_LDFLAGS="$XBPS_LDFLAGS")
	else
		_configure_args+=(HOST_ARCHITECTURE_CFLAGS="$CFLAGS"
		 HOST_ARCHITECTURE_CXXFLAGS="$CXXFLAGS" HOST_ARCHITECTURE_LDFLAGS="$LDFLAGS")
	fi

	make config "${_configure_args[@]}"
```

Reply:
also, could `HOST_{CC,CXX,LD}` and `ARCHITECTURE` be passed always?

Reply:
thanks for pointing out the use of bash arrays!

This is much cleaner